lib/ex_debug_toolbar.ex

```diff
@@ -112,7 +112,6 @@ defmodule ExDebugToolbar do
   Returns `func` return value.
   """
   @spec record_event(id, String.t(), function()) :: any()
-  @decorate noop_when_toolbar_disabled()
```

Comment:
wrapped function gets executed and results are correctly returned from this call, while toolbar related functions that wrap are still noop.